jekyll/_cci2/build-config-cookbook.md

```diff
@@ -0,0 +1,334 @@
+
+```yaml
+version: 2.1
+jobs:
+  publish-and-rollout-image:
+    description: "Update cluster with new Docker image."
+    machine: true
+    parameters:
+      deployment:
+        description: "The Kubernetes deployment name."
+        type: string
+      container:
+        description: "The Kubernetes container name."
+        type: string
+      gcloud-service-key:
+        description: The gcloud service key
+        type: env_var_name
+        default: GCLOUD_SERVICE_KEY
+      google-project-id:
+        description: The Google project ID to connect with via the gcloud CLI
+        type: env_var_name
+        default: GOOGLE_PROJECT_ID
+      google-compute-zone:
+        description: The Google compute zone to connect with via the gcloud CLI
+        type: env_var_name
+        default: GOOGLE_COMPUTE_ZONE
+      registry-url:
+        description: The GCR registry URL from ['', us, eu, asia].gcr.io
+        type: string
+        default: gcr.io
+      image:
+        description: A name for your docker image
+        type: string
+      tag:
+        description: A docker image tag
+        type: string
+        default: "latest"
+      path-to-dockerfile:
+        description: The relative path to the Dockerfile to use when building image
+        type: string
+        default: "."
+    steps:
+      - checkout
+      - gcr/gcr-auth:
+          google-project-id: <<parameters.google-project-id>>
+          google-compute-zone: <<parameters.google-compute-zone>>
+      - install
+      - gcr/build-image:
+          registry-url: <<parameters.registry-url>>
+          google-project-id: <<parameters.google-project-id>>
+          image: <<parameters.image>>
+          tag: << parameters.tag >>
+          path-to-dockerfile: <<parameters.path-to-dockerfile>>
+      - gcr/push-image:
+          registry-url: <<parameters.registry-url>>
+          google-project-id: <<parameters.google-project-id>>
+          image: <<parameters.image>>
+          tag: <<parameters.tag>>
+      - rollout-image:
+          deployment: "<<parameters.deployment>>"
+          container: "<<parameters.container>>"
+          image: "<<parameters.image>>"
+```
+
+## Example GKE Orb
+
+The example below illustrates how you can use the CircleCI GKE orb to log into the Google Cloud Platform (GCP), build and publish a docker image, and roll the image out to the GKE cluster.
+
+```yaml
+example:
+  publish-and-rollout-image:
+    description: |
+      "The simplest example of using this Orb. Logs into GCP, builds and 
+      publishes a Docker image, and then rolls the image out to a GKE cluster."
+    usage:
+      version: 2.1
+      orbs:
+        gke: felicianotech/test-gke@dev:testing-3
+      workflows:
+        main:
+          jobs:
+            - build
+            - gke/publish-and-rollout-image:
+                context: gcp-testing
+                deployment: orb-badge-server
+                image: orb-badge-server
+                tag: "2"
+```
```

Comment:
This whole piece is how orbs show their examples. Someone can't actually use the example key in their config and have this work correctly.

Reply:
Removed "example" key from this example yaml